stan/math/rev/mat/fun/gp_periodic_cov.hpp

```diff
@@ -257,7 +260,7 @@ class gp_periodic_cov_vari<T_x, double, T_l, T_p> : public vari {
         sin_2_dist_[pos] = sin(2.0 * pi_div_p * dist);
         sin_dist_sq_[pos] = sin_dist_sq;
         cov_lower_[pos] = new vari(
-            sigma_sq_d_ * std::exp(sin_dist_sq * neg_two_inv_l_sq), false);
```

Comment:
I'm OK weither way with this,  but if everything's guaranteed to be `int` or `double`, there's no reason not to directly qualify the function.

Reply:
It feels like there's a reason i did this but I can't remember. I'll change it back and if something breaks we'll know what's up